src/installer/managed/Microsoft.NET.HostModel/PEOffsets.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.NET.HostModel;
+
+/// <summary>
+/// Offsets and constants of PE file. see https://learn.microsoft.com/windows/win32/debug/pe-format
+/// </summary>
+internal static class PEOffsets
+{
+    private const int PESignatureSize = sizeof(int);
+    private const int CoffHeaderSize = 20;
+    // Signature of DOS Stub.
+    public const ushort DosStubSignature = 0x5A4D;
```

Comment:
nit: `DosStubSignature` -> `DosSignature`

The signature is for the image, not specifically for the stub.